scrape/apps.go

```diff
@@ -133,9 +134,28 @@ type AppManifest struct {
 	DefaultPermissions *github.InstallationPermissions `json:"default_permissions,omitempty"`
 }
 
-// CreateApp creates a new GitHub App with the given manifest configuration.
-func (c *Client) CreateApp(m *AppManifest) (*http.Response, error) {
-	u, err := c.baseURL.Parse("/settings/apps/new")
+type OwnerType string
+
+const (
+	OwnerTypePersonal     OwnerType = "personal"
+	OwnerTypeOrganization OwnerType = "organizational"
+)
+
+type AppConfig struct {
+	// OwnerType specifies the type of GitHub account owner (personal or organizational).
+	OwnerType OwnerType
+	// OrgName is the name of the organization if the OwnerType is organizational.
+	OrgName string
+}
+
+func (c *Client) CreateApp(m *AppManifest, ac *AppConfig) (*http.Response, error) {
```

Comment:
This AppConfig struct feels over-engineered.  In the case of creating a personal app, you can't provide a nil AppConfig, since line 154 will panic, but otherwise none of the values get used.  I *think* all you really want is the organization name, right?  So why not just pass an `orgName string` parameter, and if it's an empty string then create a personal app?

Reply:
You're right, I'll do that instead .